.github/workflows/build-all-platforms.yml

```
@@ -75,6 +75,16 @@ jobs:
     # Run this notifier only for single-platform flows; Build All/Publish are handled by the publish job
     if: ${{ always() && (github.event.inputs.flow == 'MacOS' || github.event.inputs.flow == 'Windows' || github.event.inputs.flow == 'Linux' || github.event.inputs.flow == 'Sign Windows') }}
     steps:
+      - name: Check out notification scripts
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)